java/fury-core/src/main/java/org/apache/fury/config/Config.java

```diff
@@ -38,6 +38,7 @@ public class Config implements Serializable {
   private final boolean basicTypesRefIgnored;
   private final boolean stringRefIgnored;
   private final boolean timeRefIgnored;
+  private final boolean copyTrackingRef;
```

Comment:
If we reuse ref resolver, this option can be omitted

Reply:
I am considering whether it is necessary to handle the circular references of `copy` and `serialize/deserialize` separately